VERSION

```diff
@@ -0,0 +1 @@
+3.3.0
```

Comment:
Do we need to handle there being a newline at the end of the file in CMake and MATLAB?

Reply:
It won't build because CMake won't allow new-line characters in its version string. We might need to handle it on the Matlab side though

Reply:
VERSION with a newline and trailing spaces will be stored in Git at some point, so we should be able to handle it if possible.

CMake has a STRIP command (`string(STRIP <string> <output variable>)`) which should let anything that is read be cleaned.